src/index.html

```diff
@@ -17,6 +17,61 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="link-logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML, such as between the `<header>` and the `<nav>`.